src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -437,6 +437,7 @@ private static boolean isAnonymousClassDef(DetailAST ast) {
      * or null otherwise.
      * @param ast IDENT ast to check.
      * @return the class frame where violation is found or null otherwise.
+     * @noinspection IfStatementWithIdenticalBranches
```

Comment:
@ps-sp Why do you say this is what caused wrecker failure?
@romani It looks like wrecker just timed out.

https://app.wercker.com/checkstyle/checkstyle/runs/build/58c30ebf59aaca0100b35287?step=58c30ed378bfa50001ccd5fb
> NoExceptiontest - Checkstyle ,sevntu-checkstyle
Command timed out after no response